src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -38,120 +42,131 @@ void tearDown() {
     }
 
     @Test
-    void login_Null_NotOk() {
+    void register_nullLogin_NotOk() {
```

Comment:
rename as `register_nullLogin_notOk` (`not` should start witch small letter). fix the names according the correct convention in all test methdos.